app/src/components/mapTileLayers.spec.js

```diff
@@ -0,0 +1,168 @@
+import { store } from "../store";
+import { MapTileLayers } from "./mapTileLayers";
+import { SearchResultMap } from "./searchResultMap";
+import { logException } from "../utils/logging";
+const d3Tile = require("d3-tile");
+
+jest.mock("../store");
+jest.mock("../utils/logging");
+
+jest.mock("d3-tile");
+d3Tile.tile.mockImplementation(() => {
+  const fn = jest.fn();
+  fn.tile = jest.fn().mockReturnThis();
+  fn.size = jest.fn().mockReturnThis();
+  fn.scale = jest.fn().mockReturnThis();
+  fn.translate = jest.fn().mockReturnThis();
+  fn.mockImplementation(() =>
+    Object.assign(
+      [
+        [1204, 1539, 12],
+        [1205, 1539, 12],
+        [1206, 1539, 12],
+        [1207, 1539, 12],
+        [1204, 1540, 12],
+        [1205, 1540, 12],
+        [1206, 1540, 12],
+        [1207, 1540, 12],
+      ],
+      { scale: 256, translate: [-1204.9136777777776, -1539.4919590398038] }
+    )
+  );
+  return fn;
+});
+
+describe("MapTileLayers", () => {
```

Comment:
could add a test that `init` calls `logException` when `fetchCartoTilesSchema` fails